pubtools/_quay/container_image_pusher.py

```diff
@@ -285,6 +286,12 @@ def push_container_images(self):
                 )
             except ManifestTypeError:
                 source_ml = None
+            # some registries can return 404 instead of v2ch2
+            except requests.exceptions.HTTPError as e:
```

Comment:
This is not mentioned in the commit message

Reply:
fixed